tests/test_shell.py

```diff
@@ -185,6 +185,22 @@ def test_view(self):
                 "view/a/{}/job".format(sp["a"])
             ) == os.path.realpath(project.open_job(sp).path)
 
+    @pytest.mark.skipif(WINDOWS, reason="Symbolic links are unsupported on Windows.")
+    def test_view_prefix(self):
+        self.call("python -m signac init my_project".split())
```

Comment:
Project names were removed on `next`, so this will need to be updated:
```suggestion
        self.call("python -m signac init".split())
```